app/client/cli/flags/flags.go

```diff
@@ -0,0 +1,13 @@
+package flags
+
+var (
+	RemoteCLIURL string
```

Comment:
Do these need to be exposed outside of the `flags` package?

Reply:
Yes, all flag vars must be exported in order to be used as they have been moved into their own module.